com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,238 @@
+{
+  "app-id": "com.openmusicplayer.OMP",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--share=network",
+    "--filesystem=host",
+    "--socket=pulseaudio",
+    "--socket=wayland"
+  ],
+  "cleanup": [
```

Comment:
It's definitely not enough. There are leftovers in `/app/share/pkgconfig`, `/app/share/mm-common`, `/app/share/gtk-doc`, `/app/share/aclocal` `/app/lib/gtkmm-3.0`, `/app/lib/pangomm-1.4`, `/app/./lib/gstreamer-1.0/libgstkms.la` … and so on.

```
flatpak run --command=find com.openmusicplayer.OMP /app 
```

Reply:
How do I go about finding all of the leftovers?

Reply:
@TheWiseNoob 

> How do I go about finding all of the leftovers?

```
flatpak-builder --run "build" "com.openmusicplayer.OMP.json" "sh"
```

Then just search the `/app` location, e.g. `ls -la /app`, `ls -la /app/share`, `find /app -type f -name '*.la'`. etc.